CONTRIBUTING.md

```diff
@@ -36,3 +36,41 @@ Here are a few things you can do that will increase the likelihood of your pull
 - [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/)
 - [Using Pull Requests](https://help.github.com/articles/about-pull-requests/)
 - [GitHub Help](https://help.github.com)
+
+## Directory structure
+```
+virtual-environments
+├──docs
+├──help
+├──helpers
+├──images.CI
+|   ├──/azure-pipelines 
+|   └──...
+└──images
+    ├──/win
+    |   ├──/toolsets 
+    |   ├──/Installers
+    |   |   └──...
+    |   ├──/Tests
+    |   └──/templates
+    ├──/macos
+    └──/linux
+        ├──/toolsets 
+        ├──/scripts
+        |   └──...
+        └──/config
+
+```
+- `help` - contains project documentation.
+- `helpers` - contains scripts related to image deployment.
+- `images.CI` - contains scripts that are used in image-generation CI definitions. `/azure-pipelines` folder contains YAML defintions for CI pipelines.
```

Comment:
Good change, but I suggest to introduce it separately in scope of issue where we will add documentation for Pester

Reply:
and typo, should be `definitions`

Reply:
I don't think that we should include the whole folder tree to `contributing.md` guide. In #1295 , I have added description of `└──images` folder to contributing.md (just detailed guide for every OS).
As for the root folders, like help, helpers, images.CI, I think it is better to give links to them from root readme.md